packages/blockchain-wallet-v4-frontend/src/components/Brokerage/WithdrawalLockHold.tsx

```diff
@@ -51,8 +51,8 @@ export const OnHold = (props: Props) => {
           </TooltipHost>
           <Tooltip id='onhold_tooltip' place='bottom'>
             <FormattedMessage
-              id='modals.brokerage.withdraw_holding_period'
-              defaultMessage='Newly added funds are subject to a holding period. You can transfer between your Trading, Rewards, Staking, and Exchange accounts in the meantime.'
+              id='modals.brokerage.withdraw_holding_period_1'
```

Comment:
Out of interest, do we have a process to update existing translation other than changing id?